src/libcore/mem.rs

```diff
@@ -1039,6 +1039,14 @@ impl<T> MaybeUninit<T> {
     /// Note that dropping a `MaybeUninit` will never call `T`'s drop code.
     /// It is your responsibility to make sure `T` gets dropped if it got initialized.
     #[unstable(feature = "maybe_uninit", issue = "53491")]
+    #[cfg(not(stage0))]
+    pub const fn zeroed() -> MaybeUninit<T> {
```

Comment:
This needs to have a `rustc_const_unstable` attribute as it does not adher to `min_const_fn`. Otherwise, when trying to stabilize `maybe_uninit`, you'll start getting errors.

Reply:
Sorry, I overlooked the no-`unsafe` requirement for `min_const_fn`. Added `rustc_const_unstable` and gated it behind `const_maybe_uninit_zeroed`.